packages/editor-ui/src/views/NodeView.vue

```diff
@@ -385,6 +393,7 @@ export default mixins(
 				pullConnActive: false,
 				dropPrevented: false,
 				renamingActive: false,
+				showCreateMenu: false,
```

Comment:
I was going for a more generic name here if something else is added in the future, but since it's referred as a Sticky button/menu everywhere else, I agree.